core/src/main/scala/akka/kafka/internal/TransactionalSources.scala

```diff
@@ -139,22 +144,36 @@ private[internal] abstract class TransactionalSourceLogic[K, V, Msg](shape: Sour
 
   override val groupId: String = consumerSettings.properties(ConsumerConfig.GROUP_ID_CONFIG)
 
+  private val onTransactionAbortedCb = getAsyncCallback[Unit] { _ =>
+    log.debug(s"Committing failed, draining: $draining, resetting inFlightRecords: $inFlightRecords")
```

Comment:
```suggestion
    log.debug("Committing failed, draining: {}, resetting inFlightRecords: {}", draining, inFlightRecords")
```

Is this the only use of `draining`?

Reply:
Yes.  It was for troubleshooting.  I don't think it's very useful anymore.  I'll remove it.